### PR TITLE
License date updates

### PR DESCRIPTION
**Pull request by wiktorn**

### CODE REVIEW

File: LICENSE-2.0 [hunk 1]
```diff
@@ -187,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2017 CodiLime
```
Comment (koriakin) [inline LICENSE-2.0]: Tego nie powinniśmy dotykać.

File: include/data/bindata.h [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2016 CodiLime
+ * Copyright 2017 CodiLime
```
Comment (koriakin) [inline include/data/bindata.h]: 2016-2017 proponuję

**Comment by wiktorn** [comment]
Updated

**Comment by koriakin** [comment]
Jeszcze literówka w commit msg, poza tym OK

File: python/srv [hunk 3]
```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python3
 
-# Copyright 2017 CodiLime
+# Copyright 2016-2017 CodiLime
```
Comment (koriakin) [inline python/srv]: To 2016 tutaj to akurat nieprawda...

Comment (altran01) [inline python/srv]: Up - do we want to have copyright notices for whole duration of our project on all files - not starting when file was introduced?

Comment (wiktorn) [inline python/srv]: There are two reasonable options:
1. Set the dates based on creation `date` - `last change of file`
2. Set the dates - everywhere the same 2016 - `last commit to repository`

As this repo history started in 2017, then either we drop 2016, or change all 2017 with 2016.

Manual updates are not an option IMO

**Pull request closed without merging**